src/apps/vpn/connectionbenchmark/connectionbenchmark.cpp

```diff
@@ -51,6 +54,9 @@ void ConnectionBenchmark::setConnectionSpeed() {
     m_speed = SpeedSlow;
   }
 
+  emit GleanDeprecated::instance()->recordGleanEvent(GleanSample::speedTestCompleted);
```

Comment:
what about errors? Do we care?

Reply:
Santiago and I (and Santiago and other people) have gone over [his desired metrics](https://mozilla-hub.atlassian.net/wiki/spaces/SECPRV/pages/117309748/Metrics+Collection+Dashboards+PRD) a lot. Error has not come up as something helpful to his future analysis - but if we need it, we could probably extrapolate from `completed - started`, or add a future metric.

Reply:
This has now been updated, and we're now tracking errors and cancellations.

Reply:
@bakulf  I'm going to resolve this and merge on Monday, unless you speak up and ask to do another review. (I'd love to have another review if you want, but Bea has also spent a bunch of time on this one - very much up to you.)